device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClientConfig.java

```diff
@@ -38,6 +39,10 @@
     private boolean useWebsocket;
     private ProxySettings proxySettings;
 
+    @Getter
+    @Setter(AccessLevel.PROTECTED)
```

Comment:
Isn't lombok so cool! :)

Reply:
why not private?

Reply:
lombok looks cool. New learnings for me :)